runtime/metering.go

```diff
@@ -0,0 +1,59 @@
+package runtime
+
+type MetringOperationType uint
+
+const (
+	// base [0-10)
```

Comment:
Is there any reason for leaving gaps? What if in the future the gaps are filled but more elements need to be added to a "category"?

Reply:
yeah you're right, changed it